blocks/card/card.css

```diff
@@ -0,0 +1,36 @@
+.card {
+    overflow: hidden;
+    height: auto;
+}
+
+.card a {
+    text-decoration: none;
+    display: inline-block;
+    max-width: 100%;
+    overflow: hidden;
+}
+
+.card img {
+    transition: all .65s ease;
+}
+
+.card:hover img {
+    transform: scale(1.05);
+    transform-style: preserve-3d;
+}
+
+.card h3 {
+    font-size: var(--body-font-size-s);
+    color: var(--red);
+    font-family: var(--body-font-family);
+    font-weight: var(--font-weight-medium);
+}
+
+.card .card-body p {
+    font-weight: var(--font-weight-normal);
+    color: var(--charcoal-blue);
+}
+
+.card .card-body p:hover {
+    color: var(--maroon);
```

Comment:
Move this hover color to the same "body.red" styles

Reply:
done